crates/net/rpc/src/eth/api/transactions.rs

```diff
@@ -43,3 +43,110 @@ where
         Ok(hash)
     }
 }
+
+
+#[cfg(test)]
+mod tests {
+    use reth_transaction_pool::test_utils::testing_pool;
+    use reth_primitives::Bytes;
+
+    use crate::EthApi;
+    use test_utils::NoopClient;
+    use std::{sync::Arc, str::FromStr};
+
+
+
+    #[tokio::test]
+    async fn send_raw_transaction() {
+
+        let pool = testing_pool();
+
+        let client = NoopClient::default();
+
+        let eth_api = EthApi::new(Arc::new(client), pool.clone(), ());
+
+        let tx = Bytes::from_str("0x123456789abcdef").unwrap();
+
+        eth_api.send_raw_transaction(tx).await.unwrap();
+
+        assert!(!pool.is_empty());
+
+    }
+
+
+
+    mod test_utils {
+        use reth_provider::{BlockProvider, StateProviderFactory, BlockHashProvider, StateProvider, AccountProvider};
+        use reth_interfaces::Result;
+        use reth_primitives::{ChainInfo, U256, rpc::BlockId, Block};
+
+        #[derive(Debug, Default)]
+        pub(crate) struct NoopClient;
+
+        pub(crate) struct NoopSP;
+
```

Comment:
I see that we need some additional trait impls.

I'd suggest that we add these directly to:
 https://github.com/paradigmxyz/reth/blob/ba70b3b7ef7776770cb675f2cb9fd927bb5646be/crates/storage/provider/src/test_utils/noop.rs#L8